### ISSUE TITLE
Error to create a bridge network

### ISSUE THREAD

**Issue by jrziviani** [kimchi-project/kimchi#781]
I was not able to create a bridge network due to the following error:
```
KCHNET0020E: Failed to activate interface : Usage: ip link add [link DEV] [ name ] NAME [ txqueuelen PACKETS ] [ address LLADDR ] [ broadcast LLADDR ] [ mtu MTU ] [ numtxqueues QUEUE_COUNT ] [ numrxqueues QUEUE_COUNT ] type TYPE [ ARGS ] ip link delete DEV type TYPE [ ARGS ] ip link set { dev DEVICE | group DEVGROUP } [ { up | down } ] [ arp { on | off } ] [ dynamic { on | off } ] [ multicast { on | off } ] [ allmulticast { on | off } ] [ promisc { on | off } ] [ trailers { on | off } ] [ txqueuelen PACKETS ] [ name NEWNAME ] [ address LLADDR ] [ broadcast LLADDR ] [ mtu MTU ] [ netns PID ] [ netns NAME ] [ alias NAME ] [ vf NUM [ mac LLADDR ] [ vlan VLANID [ qos VLAN-QOS ] ] [ rate TXRATE ] ] [ spoofchk { on | off} ] ] [ state { auto | enable | disable} ] ] [ master DEVICE ] [ nomaster ] ip link show [ DEVICE | group GROUP ] [up] TYPE := { vlan | veth | vcan | dummy | ifb | macvlan | macvtap | can | bridge | ipoib | ip6tnl | ipip | sit | vxlan | gre | gretap | ip6gre | ip6gretap | vti } .
```
After a quick debug I saw the the network interface destination is not being received by the backend code:
```python
    def _set_network_bridge(self, params):
        try:
            # fails if host interface is already in use by a libvirt network
            iface = params['interface']
            ...
```
```
(Pdb) p params
{u'interface': u'', 'forward': {'mode': u'bridge'}, u'connection': u'bridge', u'name': u'my_br'}
```

**Comment by jrziviani** [comment]
I'm working on it.

**Comment by alinefm** [comment]
Fixed by 5a88a9539422a0e485a19f896f620add5e9f551f
